Task_DEV_5/Task_DEV_5/BubbleSorter.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System;
+using System.Text;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class BubbleSorter
+    {
+        private string path;
+        public FileStream fstream;
+
+        public BubbleSorter(string path)
+        {
+            this.path = path;
+        }
+
+        private string ReadLine()
```

Comment:
Private methods should be one by one after public methods.

Reply:
Done.